esp/esp/themes/theme_data/fruitsalad/templates/main.html

```diff
@@ -107,10 +107,23 @@
 	</form>
       </div>
       <div id="contact_info">
-	<span class="title accentcolor">{{ theme.full_group_name }}</span>
-      <span class="normal">{{ theme.contact_info|linebreaksbr }}</span>
-      <span class="normal">E-mail: <a href="mailto:{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}">{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}</a></span>
-      <span class="normal"><a href="/faq.html">FAQ</a> | <a href="/contact.html">contact us</a>{% if theme.facebook_link %} | <a href="{{ theme.facebook_link }}">facebook</a>{% endif %}</span>
+        <span class="title accentcolor">{{ theme.full_group_name }}</span>
+        <span class="normal">{{ theme.contact_info|linebreaksbr }}</span>
+        <span class="normal">E-mail: <a href="mailto:{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}">{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}</a></span>
+        <span class="normal">
+          {% if theme.faq_link == None %}
+            {# If faq_link is undefined, fall back to the old hardcoded value.
```

Comment:
This is beside the point since I just suggested deleting it, but multi-line comments need different syntax: https://docs.djangoproject.com/en/1.8/ref/templates/language/#comments